.ci/lint-powershell.ps1

```diff
@@ -53,4 +53,4 @@ $settings = @{
     }
 }
 
-Invoke-ScriptAnalyzer -Path "$env:BUILD_DIRECTORY/.ci" -Recurse -EnableExit -Settings $settings
+Invoke-ScriptAnalyzer -Path ./ -Recurse -EnableExit -Settings $settings

```

Comment:
Just for the consistency with other our linting scripts.
https://github.com/microsoft/LightGBM/blob/92aa07b4b827d020d9aeeeddec0b3416042c9d2a/.ci/lint-cpp.sh#L6-L9
https://github.com/microsoft/LightGBM/blob/92aa07b4b827d020d9aeeeddec0b3416042c9d2a/.ci/lint-python.sh#L10-L12

Reply:
Thanks! I agree, I think this makes the scripts a little easier to read.